pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -1794,3 +1811,75 @@ func isDeployDB(dbType api.DatabaseType) bool {
 	}
 	return false
 }
+
+func getCORSConfig(cors *api.CORSFiltersObject) {
+	if cors != nil {
+
+		//Full cors.Filters = "AC_ALLOW_ORIGIN,AC_ALLOW_METHODS,AC_ALLOW_HEADERS,AC_ALLOW_CREDENTIALS,AC_MAX_AGE"
```

Comment:
keep in mind that, user, besides to have the option to set custom filters dynamically he can just enable CORS with all default configuration.

Reply:
Ok, I'll add a checkbox to enable CORS with default